go/contracts/go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/chain4travel/camino-messenger-contracts/go/contracts
 
-go 1.23.5
+go 1.23.6
```

Comment:
is it important to make this change ? can we just make it 1.23 ?

Reply:
That file is created by the `abigen` tool. And afaik it uses what ever Go version it's running on. 

On the other hand, I think it's important to make that change as .6 patch release is a [security-fix release](https://go.dev/doc/devel/release#go1.23.0) for `crypto/elliptic` package.